src/main/java/goral/psychotherapistoffice/domain/meeting/MeetingService.java

```diff
@@ -36,13 +42,17 @@ public MeetingService(PatientRepository patientRepository, CalenderRepository ca
     @Transactional
     public void addMeeting(MeetingToSaveDto meetingToSaveDto){
         Meeting meeting = new Meeting();
-        Patient patient = patientRepository.findPatientById(meetingToSaveDto.getPatient()).orElseThrow();
+        Patient patient = patientJpaRepository.findPatientById(meetingToSaveDto.getPatient()).orElseThrow();
         meeting.setPatient(patient);
         Therapy therapy = therapyRepository.findById(meetingToSaveDto.getTherapy()).orElseThrow();
         meeting.setTherapy(therapy);
-        Calender calender = calenderRepository.findById(meetingToSaveDto.getCalender()).orElseThrow();
-        meeting.setCalender(calender);
-        calender.setFree(false);
+        Calender calender = calenderRepository.findCalenderByIdAndFreeIsTrue(meetingToSaveDto.getCalender()).orElseThrow();
```

Comment:
i think we can set name little easier, `findCalenderByIdAndIsFree`. for negation we can set `findCalenderByIdAndIsNotFree`